k8s-operator/apis/v1alpha1/types_connector.go

```diff
@@ -156,6 +156,15 @@ type ConnectorStatus struct {
 	// IsExitNode is set to true if the Connector acts as an exit node.
 	// +optional
 	IsExitNode bool `json:"isExitNode"`
+	// TailnetIPs is the set of tailnet IP addresses (both IPv4 and IPv6)
+	// assigned to the Connector node(s).
+	// +optional
+	TailnetIPs []string `json:"tailnetIPs,omitempty"`
+	// Hostname is the fully qualified domain name of the Connector node(s).
+	// If MagicDNS is enabled in your tailnet, it is the MagicDNS name of the
+	// node.
+	// +optional
+	Hostname string `json:"hostname,omitempty"`
```

Comment:
Are strings ok for these types? I was tempted to use the `Hostname` type for the second one, but that mostly seems to be for the purpose of spec validation rules.

Reply:
> Are strings ok for these types?

Yes they are. And yes, the `Hostname` type is just for validation of the hostname field that user sets, we shouldn't need to do it for status that we set